ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitter.scala

```diff
@@ -497,9 +497,10 @@ private[kvutils] class TransactionCommitter(
     }
     // Update contract keys.
     val ledgerEffectiveTime = transactionEntry.submission.getLedgerEffectiveTime
-    for ((key, contractKeyState) <- effects.updatedContractKeys) {
+    contractKeys.foreach { case (contractKey, contractKeyState) =>
```

Comment:
Same here regarding for comprehension:
```suggestion
    for ((contractKey, contractKeyState) <- contractKeys) {
```